test/spec/unit/core/bidderFactory_spec.js

```diff
@@ -1456,67 +1456,85 @@ describe('bidderFactory', () => {
-        let fledgeStub;
 
-        function fledgeHook(next, ...args) {
-          fledgeStub(...args);
+      it('should unwrap bids', function() {
+        const bidder = newBidder(spec);
+        spec.interpretResponse.returns({
+          bids: bids,
+        });
+        bidder.callBids(bidRequest, addBidResponseStub, doneStub, ajaxStub, onTimelyResponseStub, wrappedCallback);
+        sinon.assert.calledWith(addBidResponseStub, 'mock/placement', sinon.match(bids[0]));
+      });
+
+      it('does not unwrap bids from a bid that happens to have a "bids" property', () => {
+        const bidder = newBidder(spec);
+        const bid = Object.assign({
+          bids: ['a', 'b']
+        }, bids[0]);
+        spec.interpretResponse.returns(bid);
+        bidder.callBids(bidRequest, addBidResponseStub, doneStub, ajaxStub, onTimelyResponseStub, wrappedCallback);
+        sinon.assert.calledWith(addBidResponseStub, 'mock/placement', sinon.match(bid));
+      })
+
+      describe('when response has PAAPI auction config', function() {
+        let paapiStub;
+
+        function paapiHook(next, ...args) {
+          paapiStub(...args);
         }
 
         before(() => {
-          addComponentAuction.before(fledgeHook);
+          addComponentAuction.before(paapiHook);
         });
 
         after(() => {
-          addComponentAuction.getHooks({hook: fledgeHook}).remove();
+          addComponentAuction.getHooks({hook: paapiHook}).remove();
         })
 
         beforeEach(function () {
-          fledgeStub = sinon.stub();
-        });
-
-        it('should unwrap bids', function() {
-          const bidder = newBidder(spec);
-          spec.interpretResponse.returns({
-            bids: bids,
-            fledgeAuctionConfigs: []
-          });
-          bidder.callBids(bidRequest, addBidResponseStub, doneStub, ajaxStub, onTimelyResponseStub, wrappedCallback);
-          expect(addBidResponseStub.calledOnce).to.equal(true);
-          expect(addBidResponseStub.firstCall.args[0]).to.equal('mock/placement');
+          paapiStub = sinon.stub();
         });
 
-        it('should call fledgeManager with FLEDGE configs', function() {
-          const bidder = newBidder(spec);
-          spec.interpretResponse.returns({
-            bids: bids,
-            fledgeAuctionConfigs: [fledgeAuctionConfig]
-          });
-          bidder.callBids(bidRequest, addBidResponseStub, doneStub, ajaxStub, onTimelyResponseStub, wrappedCallback);
+        ['fledgeAuctionConfigs', 'paapiAuctionConfigs'].forEach(paapiProp => {
+          describe(`using ${paapiProp}`, () => {
+            it('should call paapi hook with PAAPI configs', function() {
+              const bidder = newBidder(spec);
+              spec.interpretResponse.returns({
+                bids: bids,
+                [paapiProp]: [paapiConfig]
+              });
+              bidder.callBids(bidRequest, addBidResponseStub, doneStub, ajaxStub, onTimelyResponseStub, wrappedCallback);
```

Comment:
maybe a test for `prefers 'paapiAuctionConfigs' over 'fledgeAuctionConfigs' if both defined`?

Prob not a real life scenario though so maybe not!